domain/schema/controller/sql/0009-controller-node.sql

```diff
@@ -9,3 +9,15 @@ ON controller_node (dqlite_node_id);
 
 CREATE UNIQUE INDEX idx_controller_node_bind_address
 ON controller_node (bind_address);
+
+-- controller_node_agent_version tracks the reported agent version running for
+-- each controller in the cluster.
+CREATE TABLE controller_node_agent_version (
+    controller_id TEXT NOT NULL PRIMARY KEY,
+    version TEXT NOT NULL,
```

Comment:
We talked about breaking out "tag" and "build" as a separate column and keeping this as the semver column.
I still think we will eventually end up wanting to do that but we can change later if/when needed.

Reply:
I am inclined to agree that we should prove out we are going to need this. I think for tracking agent version in the object store we will need it. But this is just a reporting table that is read only 100% of the time.